remote/remote_test.go

```diff
@@ -1595,5 +1658,47 @@ func testImage(t *testing.T, when spec.G, it spec.S) {
 				h.AssertEq(t, image.CheckReadWriteAccess(), false)
 			})
 		})
+
+		when("using custom handler", func() {
+			it.Before(func() {
+				os.Setenv("DOCKER_CONFIG", customRegistry.DockerDirectory)
+			})
+
+			it.After(func() {
+				os.Setenv("DOCKER_CONFIG", dockerRegistry.DockerDirectory)
+			})
+
+			when("image has read/write access", func() {
```

Comment:
```suggestion
			when("image has read and write access", func() {
```

Reply:
Added in the following [commit](https://github.com/buildpacks/imgutil/pull/125/commits/87d5ba9fc355d30923e2cf672f08e02c9de908fd)